### ISSUE TITLE
Fix defaults function

### ISSUE THREAD

**Issue by devoid** [ModelSEED/ModelSEED#90]
`ms defaults` originally allowed the user to:
1. select "default" biochemistry, mapping, model object to use in the event that the user didn't supply one.
2. select a "defaultNameSpace" for biochemistry, mapping and model?

Niether of these work now. I don't think we _can_ do 1; it would be a lot of work and the code for parsing CLI arguments is a pain already.  2 is doable, but requires some work since we don't currently check 
`ModelSEED::Configuration` when looking at a "defaultNameSpace" attribute.

An alternative is to re-do the `ms default` function, possibly renaming it, to allow the user to switch alias sets. I'm thinking:

``` bash
$ bio defaultNameSpace biochemistry/sdevoid/main
ModelSEED
$ bio defaultNameSpace biochemistry/sdevoid/main KEGG
$
```

**Comment by cshenry** [comment]
Yeah, I say delete that function, and if we so desire, add a "setnamespace" command to "model" and  "biochemistry". I don't think it's appropriate to even make it a mseed command, as it only applies to two provenance objects.
